packages/http-client-java/generator/http-client-generator-test/src/main/java/com/parameters/bodyoptionality/BodyOptionalityAsyncClient.java

```diff
@@ -125,8 +123,8 @@ public Mono<Void> requiredExplicit(BodyModel body) {
     public Mono<Void> requiredImplicit(String name) {
         // Generated convenience method for requiredImplicitWithResponse
         RequestOptions requestOptions = new RequestOptions();
-        RequiredImplicitRequest requiredImplicitRequestObj = new RequiredImplicitRequest(name);
-        BinaryData requiredImplicitRequest = BinaryData.fromObject(requiredImplicitRequestObj);
-        return requiredImplicitWithResponse(requiredImplicitRequest, requestOptions).flatMap(FluxUtil::toMono);
+        BodyModel bodyModelObj = new BodyModel(name);
+        BinaryData bodyModel = BinaryData.fromObject(bodyModelObj);
+        return requiredImplicitWithResponse(bodyModel, requestOptions).flatMap(FluxUtil::toMono);
```

Comment:
These changes is due to TCGC change on spread.

Now it does not create a new anonymous model (`RequiredImplicitRequest`), but reuse existing model `BodyModel`.